digital-anonymity.md

```diff
@@ -0,0 +1,161 @@
+
+
+## Communication Tools and Safeguarding Freedom of Speech:
+
+
+Virtual Private Networks (VPNs) can encrypt internet traffic. Numerous messaging systems like Signal, Telegram, or Bison Relay (bisonrelay.org) offer an alternative to common platforms such as Whatsapp.
+
+For example, you could use a VPN to connect to the Tor browser and operate in some form of anonymity. Thinking about a future where decentralized social media platforms are available and messaging also will allow people to say how they feel without worrying they will be charged under some new law.
+
+## Future Prospects and the Way Forward:
+
+
+Balancing one's security and convenience online remains a big challenge. User-friendly solutions should be built that will seamlessly integrate with daily online interactions.
+
+The legal side of things:
+The right balance should be struck that allows users to operate online in some form of anonymity and to have an ability of freedom of speech that is protected.
+
+Education:
+Users need to be informed of their rights and what they can and cannot say or can or cannot share online. The right to know what data has been collected clearly and how to turn off what you do not wish to share.
+
+Privacy Tools:
+There may come a point when tools like VPNs, anonymous messaging, and anonymous browsers may no longer exist. These should be preserved and constantly evolve.
+
+## A Personal Perspective and Conclusion:
+
+
+The quest for anonymity continues. My perspective would be that each individual has a right to privacy online, a right to freedom of speech, and a right to transact with an anonymous digital currency if they so wish.
+
+Steps must be taken to combat government surveillance and corporations harvesting data. Personal liberties must be preserved at all costs.
+
+Control of your finances is one area that should be paramount. We have seen that we cannot trust the current financial system and the worry of holding money in the traditional banks does not come without inherent risks.
+
+When I look ahead I see great opportunities, user-friendly solutions, educated and informed users, and laws that protect users.
+
+To conclude I believe the path to digital anonymity is achievable. The commitment must come from the users. Maintaining freedom of speech, transacting, and operating online without worry. We can see this will take a collective effort but my hope through this article is to inform users of the possibility of a future where one can operate online without intrusion from governments or corporations. I hope you have enjoyed reading and I welcome any feedback.
+
```

Comment:
Grammatically I can't see any issues with this piece. I also like that you have used elements of "identify and explain" to bring in meaning, examples and personal insight.

My main criticism is this piece reads like two lists and a conclusion. e.g. here's a list of the "Challenges We Face" and here's a list of how "Crypto will solve these challenges". Which isn't necessarily a problem but the main issues I see with this kind of structure is"

* It throws a lot of points at the user. Which are relevant and structured correctly but never go into too much depth regarding a singular point.
* And secondly, because there is so much information it's easy for the reader to switch off and not engage.

One thing I do with my pieces is read them out loud or get someone else to read them to me. This give me an opportunity to get critical feedback and ask simple questions:

* Is what I've written engaging? 
* if not, what can I remove, change or edit?
* if not, which points were either repetitive, not necessary or didn't add to the piece?
* If I liked a specific point how can I build on that point to add more detail and impact?

Final note. This point should probably be identified before writing the piece. How will I tell the story? what is my writing style? e.g. persuasive, narrative, expository, and descriptive?

Examples:

Persuasive:  For this writing style, the writer is trying to convince the reader of the validity of a certain position or argument. Persuasive writing includes the writers’ opinions, and provides justifications and evidence to support their claims.

Narrative:  Often seen in longer writing samples, the purpose of this writing style is to share information in the context of a story. Narratives should include characters, conflicts, and settings.

Expository: This type of writing is used to explain a concept and share information to a broader audience. Expository writing provides evidence, statistics, or results and focuses on the facts of a certain topic. This type is not meant to express opinions.

Descriptive: This type of writing is used to depict imagery to create a clear picture in the mind of the reader. This method helps the readers become more connected to the writing by appealing to their senses. Descriptive writing employs literary techniques such as similes, metaphors, allegory, etc to engage the audience.


If you think about writing from the perspectives of one of these styles it's a little easier to see the short coming. For instance, this piece, I would say, is aiming to be "Expository" e.g. You are explaining a concept. If this is your approach the area you fall short is evidence, statistics, or result. These are the elements in this style that will give your piece the level of depth needed.

Hopefully, this info is useful. My main point here is to make the piece more engaging.

Reply:
I agree. I need to continue to work on this and as you say it is very important before I start the article to know the path it will take as you have described. I will try this with the Supply shock article which would be Expository.